space-framework/space-provider/kind/kind.go

```diff
@@ -81,8 +81,12 @@ func (k KindClusterProvider) Get(spaceName string) (clusterprovider.SpaceInfo, e
 	}
 
 	spaceInfo := clusterprovider.SpaceInfo{
-		Name:   spaceName,
-		Config: cfg,
+		Name: spaceName,
+		Config: map[string]string{
+			clusterprovider.INCLUSTER: cfg,
+			//TODO  get the incluster config
+			clusterprovider.EXTERNAL: "",
```

Comment:
If the provider has only external config we can put it in both secrets

Reply:
no. I added a field to spec